src/whylogs/core/metrics/nlp_metrics.py

```diff
@@ -0,0 +1,80 @@
+from typing import List, Union
+
+import jiwer
+
+from whylogs.core.statistics import NumberTracker
+from whylogs.proto import NLPMetricsMessage
+
+
+class NLPMetrics:
```

Comment:
Can you add some basic tests for this, especially around merge and serialization/deserialization?

Reply:
merge was added, will add one for serialization and deseriallization

Reply:
done